tests/mocha/server/HelpersTest.js

```diff
@@ -0,0 +1,24 @@
+MochaWeb.testOnly(function(){
+  describe('helpers', function() {
+
+    describe('getCompetitionEvents', function() {
+      it('gets events', function() {
+        chai.expect(getCompetitionEvents(null)).to.deep.equal([]);
+
+        var compId = Competitions.insert({ competitionName: "Comp One", listed: false, startDate: new Date() });
```

Comment:
I would prefer compId to be competitionId, even though we did change competitor to participant everywhere.